lms/services/course.py

```diff
@@ -121,6 +122,5 @@ def _is_pre_sections(self, authority_provided_id):
 def course_service_factory(_context, request):
     return CourseService(
         request.find_service(name="application_instance"),
-        request.lti_user.oauth_consumer_key,
```

Comment:
I don't think `request.lti_user.oauth_consumer_key` and request.find_service(name="application_instance").consumer_key should be ever different.